gammapy/maps/axes.py

```diff
@@ -0,0 +1,145 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+from astropy.time import Time
+
+
+class TimeAxis:
```

Comment:
Even if it doesn't inherit from `MapAxis` (yet) I think we should call it a `TimeMapAxis`